src/liballoc_jemalloc/build.rs

```diff
@@ -33,18 +33,26 @@ fn main() {
                  jemalloc.parent().unwrap().display());
         let stem = jemalloc.file_stem().unwrap().to_str().unwrap();
         let name = jemalloc.file_name().unwrap().to_str().unwrap();
-        let kind = if name.ends_with(".a") {"static"} else {"dylib"};
+        let kind = if name.ends_with(".a") {
+            "static"
+        } else {
+            "dylib"
+        };
```

Comment:
cc @nrc . In the If expression. If we return static (?) strings  and the total if expression fits  under 100 characters, then we format the code like this.
`let kind = if name.ends_with(".a") { "static" } else { "dylib" };`
what do you say?